docs/tutorial.md

```diff
@@ -99,3 +100,20 @@ Since the tool is experimental, it is better to use your intuition to come up wi
 ## Step 6: (optional) Custom Scheduling
 
 "Custom scheduling" allows you to introduce new scheduling rules on top of FSRS. This feature is only for advanced users and developers.
+
+# Add-On Compatibility
+
+Some add-ons can cause conflicts with FSRS. As a general rule of thumb, if an add-on affects a card's intervals, it shouldn't be used with FSRS.
+
+| Add-on                                                       | Compatible? | Comment |
+| ------------------------------------------------------------ |-------------------| ------- |
+|[Advanced Review Bottom Bar](https://ankiweb.net/shared/info/1136455830)|Yes✅|Please use the latest version.|
+|[The KING of Button Add-ons](https://ankiweb.net/shared/info/374005964)|Yes✅|Please use the latest version.|
+| [Pass/Fail](https://ankiweb.net/shared/info/876946123) |Yes✅| `Pass` is the equivalent of `Good`.|
+|[Incremental Reading v4.11.3 (unofficial clone)](https://ankiweb.net/shared/info/999215520)|No❌|It shows the interval given by Anki's built-in scheduler, not the custom scheduler.|
```

Comment:
I think this add-on is compatible now, because FSRS has been built-in.

Reply:
Which one? Incremental Reading? It doesn't work in 23.10, so I removed it from the table in the second commit.